tpcx_bb/cluster_configuration/bsql-cluster-startup.sh

```diff
@@ -0,0 +1,73 @@
+#IB, NVLINK, or TCP
+CLUSTER_MODE=$1
+USERNAME=$(whoami)
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="25GB"
+POOL_SIZE="30GB"
+
+# Fill in your environment name and conda path on each node
+TPCX_BB_HOME="/home/$USERNAME/shared/tpcx-bb"
+CONDA_ENV_NAME="rapids-tpcx-bb"
+CONDA_ENV_PATH="/home/$USERNAME/conda/etc/profile.d/conda.sh"
+
+# TODO: Unify interface/IP setting/getting for cluster startup
+# and scheduler file
+INTERFACE="ib0"
+
+# TODO: Remove hard-coding of scheduler
+SCHEDULER=$(hostname)
+SCHEDULER_FILE=$TPCX_BB_HOME/tpcx_bb/cluster_configuration/example-cluster-scheduler.json
+LOGDIR="/tmp/tpcx-bb-dask-logs/"
+WORKER_DIR="/tmp/tpcx-bb-dask-workers/"
+
+# Purge Dask worker and log directories
+rm -rf $LOGDIR/*
+mkdir -p $LOGDIR
+rm -rf $WORKER_DIR/*
+mkdir -p $WORKER_DIR
+
+# Purge Dask config directories
+rm -rf ~/.config/dask
+
+# Activate conda environment
+source $CONDA_ENV_PATH
+conda activate $CONDA_ENV_NAME
+
+# Dask/distributed configuration
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT="100s"
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__TCP="600s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MIN="1s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
+
+
+# Setup scheduler
+if [ "$HOSTNAME" = $SCHEDULER ]; then
+  if [ "$CLUSTER_MODE" = "NVLINK" ]; then
```

Comment:
Do we expect Blazing to work on a UCX cluster with nvlink or IB currently?

If not, perhaps we should remove them to avoid confusion.

Reply:
At the moment there is a big effort to implement UCX in the Blazingsql engine, but I don't know the timeline with precision, so, I think @williamBlazing may give us their opinion here.

Reply:
Ok. Let's leave it then. This way we can also check whether Blazing & Dask can work in the same cluster via TCP over UCX.